support.md

```diff
@@ -0,0 +1,36 @@
+# .NET Support and Distributions
+
+.NET is available (as a binary distribution) in many package managers and official download sites. These builds of .NET are provided and supported by  organizations that participate in the .NET community, [build .NET from source](https://github.com/dotnet/dotnet), and offer stable binaries to their users. These organizations work together to enable developers to be productive and safe on Linux, macOS, and Windows.
+
+Patches are provided for functional or security issues, typically monthly on [Patch Tuesday](https://en.wikipedia.org/wiki/Patch_Tuesday).
+
+.NET is supported per the following policies:
+
+* [.NET releases](releases.md)
+* [.NET release policies](release-policies.md)
+* [.NET supported operating system lifecycle](os-lifecycle-policy.md).
+
+
+Community builds are available for [Linux](linux.md) and [macOS](https://formulae.brew.sh/cask/dotnet-sdk). Community support is available on GitHub, such as at [dotnet/core](https://github.com/dotnet/core).
```

Comment:
is our repo where they would come to file issues for community builds?

Reply:
Fair point. That wasn't exactly the intent. I'm not quite sure how to word that in a simple way.